api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -336,13 +343,30 @@ private boolean shouldUserBeRegistered(DbUser user) {
         accessModuleService.isModuleCompliant(user, AccessModuleName.PUBLICATION_CONFIRMATION);
     boolean profileConfirmationComplete =
         accessModuleService.isModuleCompliant(user, AccessModuleName.PROFILE_CONFIRMATION);
+
+    Institution institution =
+        institutionService
+            .getByUser(user)
+            .orElseThrow(
+                () -> new ServerErrorException("Institution not found for the given user"));
```

Comment:
Can we not continue here? I think I would log severe, and then treat it the same as `!institutionalEmailValid`

I am slightly worried about a data issue potentially causing failures across multiple users

Reply:
I made eRARequired to true, and emailInvalid if institution not found. Not sure if a data issue may still make people kicked off from RT because of emailInvalid

Reply:
Thanks - this is great.

Right - you did the analysis already on this - do you know whether that got resolved by the admins?

Reply:
not yet. I will ask Shimon again tomorrow. If that's resolved, I will merge this tomorrow. If not, I probably need to wait after next release.